src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java

```diff
@@ -0,0 +1,84 @@
+package ch.bfh.btx8081.w2019.green.alzman.view;
+
+import com.vaadin.flow.component.accordion.Accordion;
+import com.vaadin.flow.component.accordion.AccordionPanel;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.dependency.CssImport;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.html.H4;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.listbox.ListBox;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.router.Route;
+
+/**
+ * The user management view will be used to add and remove users of the app
+ */
+@Route("")
+@CssImport(value = "./styles/shared-styles.css", include = "common-styles")
+public class InfoboxView extends TemplateView {
+
+	private H4 title1;
```

Comment:
you could use a bit more informative name declarations

Reply:
gratuliere :)